plugin/index.php

```diff
@@ -1,14 +1,18 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
  */
-
+// Hook.
```

Comment:
Are you suggesting that we should not document this? Do you have an alternate suggestion here?